Sources/Plasma/NucleusLib/pnInputCore/plInputMap.cpp

```diff
@@ -659,7 +659,7 @@ ST::string plKeyMap::GetStringUnmapped()
         default:
             break;
     }
-    return ST_LITERAL("(unmapped)");
+    return ST_LITERAL("(Unmapped)");
```

Comment:
This is inconsistent with the hardcoded uses of `(unmapped)` all over the place :/

Also, this value ends up in the `KeyMap` chronicle as far as I can tell. Are we sure that this change won't break anything when switching from/to older clients? It's *probably* fine because "unmapped" is the default case when it can't translate a string back to a `plKeyDef`, but still, I wouldn't change this unless there's a good reason...

I wouldn't worry about consistency with the other languages. The capitalization there is non-standard anyway (at least for French and German, and probably Italian and Spanish too) - they should actually all be lowercase with a space in between. I assume the CamelCase is a workaround because the `KeyMap` chronicle uses spaces as separators, so an individual key name cannot safely contain a space.

Reply:
I had to chase down all of this to make sure that things didn't break if we change languages... What calls to bind a key does is check the current language for the key. If it's not found, it goes through all the other languages, looking for that key. If it's still not found, it gives up and treats it as `plKeyCombo::kUmapped`.